OTAPI.Modifications/OTAPI.Modifications.MakeTypesPublic/OTAPI.Modifications.MakeTypesPublic.csproj

```diff
@@ -41,6 +41,9 @@
       <HintPath>..\..\packages\Mono.Cecil.0.9.6.4\lib\net45\Mono.Cecil.Rocks.dll</HintPath>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="TerrariaServer">
+      <HintPath>..\..\TerrariaServer.exe</HintPath>
```

Comment:
Hmm, so, you need to update the downloader -- you can't direct reference this from a different location. Sorry.